db/migrate/20180207234900_ba_key_stuff.rb

```diff
@@ -0,0 +1,12 @@
+class BaKeyStuff < ActiveRecord::Migration[5.0]
+	def change
+		change_column :competitions, :id, :string
```

Comment:
Well the idea was to use the BlueAlliance keys as the actual key, so that I can just have the `has_many` thing be magic. I actually can't remember anymore, but I had another reason